javascript.md

```diff
@@ -50,6 +50,14 @@ interpreted as described in [RFC 2119](http://www.ietf.org/rfc/rfc2119.txt)
 * [Function Arguments](#function-arguments)
 
 
+## Enumerables
+
+* [Enumerables](#enumerables)
+
+
+---
```

Comment:
Oh, I'm seeing why you thought we might need this.  The "Enumerables" heading should have `###` rather than `##`